docs/data/system/styled/styled.md

```diff
@@ -9,6 +9,12 @@ This utility is built on top of the `styled()` module of `@mui/styled-engine` an
 
 ### Import path
 
+```ts
+import { styled } from '@mui/system';
+// If you are using @mui/material
+import { styled } from '@mui/material/styles';
+```
+
 You can use the utility coming from the `@mui/system` package, or if you are using `@mui/material`, you can import it from `@mui/material/styles`.
```

Comment:
Can we add the snippet after the explanation?

Reply:
Docs does have `Basic usage` example here (https://deploy-preview-43223--material-ui.netlify.app/system/styled/#basic-usage)

 Are you imaging snippet to be different from `basic usage` code?

Reply:
I just meant to add it after the text